InfiniteRecharge/src/main/java/frc/robot/subsystem/vision/VelocityPoint.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystem.vision;
+
+public class VelocityPoint implements Comparable<VelocityPoint> {
```

Comment:
Just imagine how clean this class would be if everything were public variables...